etc/db-config.yaml

```diff
@@ -235,12 +239,16 @@
             type: string
             default_value: ""
             public: true
-            description: Team category for users that register themselves with the system. Self-registration is disabled if this field is left empty.
+            description: Team category for users that register themselves with the system
```

Comment:
Nitpick: I think these should be full sentences, so should end with a `.` (or `?`).
Also, I think it's best to leave the second sentence to clarify that it can be disabled that way.

Reply:
I will add the dot. The dropdown does show 'Self registration disabled' so I wonder if we should add it to the description as well...?

Reply:
Ah, ok. Then the description is fine as is.